### ISSUE TITLE
windows 11 report virus when wails build

### ISSUE THREAD

**Issue by mllw** [wailsapp/wails#3045]
### Description

wails  build
DEB | Using go webview2loader
Wails CLI v2.6.0

# Build Options

Platform(s)        | windows/amd64
Compiler           | D:\apps\Go\bin\go.exe
Skip Bindings      | false
Build Mode         | production
Devtools           | false
Frontend Directory | E:\workspace\golang\waterfaute\frontend
Obfuscated         | false
Skip Frontend      | false
Compress           | false
Package            | true
Clean Bin Dir      | false
LDFlags            |
Tags               | []
Race Detector      | false

# Building target: windows/amd64

  • Generating bindings:   ERROR  
          changeme: go build changeme: open C:\Users\jq\AppData\Local\Temp\go-build664362386\b001\exe\a.out.exe: Operation did not complete successfully because the file contains a virus or potentially unwanted software.

          exit status 1

  ERROR  
          changeme: go build changeme: open C:\Users\jq\AppData\Local\Temp\go-build664362386\b001\exe\a.out.exe: Operation did not complete successfully because the file contains a virus or potentially unwanted software.

          exit status 1

### To Reproduce

run wails build

### Expected behaviour

success build

### System Details

```shell
Microsoft Windows [版本 10.0.22621.2428]
```

**Comment by PylotLight** [comment]
Personally I've found the details in wails.json can make a difference, I'll grab my work config and do some testing and report back.

**Comment by Ggpsv** [comment]
I am running into this issue as well, out of nowhere today. Also using Windows 11.

**Comment by leaanthony** [comment]
Yes, I had it with Task as well last night. This isn't a Wails issue. This has been covered many times.

**Comment by Ggpsv** [comment]
@leaanthony So I suspected, thank you. In your experience, is it a false positive? It went away as I restarted my machine, though I did ran Defender just in case.

**Comment by leaanthony** [comment]
I can't speak for your application but in general, Go applications can get flagged by antivirus vendors because they have poor virus detection mechanisms.

**Comment by mllw** [comment]
step by step：

1、wails init -n wailstest

DEB | Using go webview2loader
Wails CLI v2.6.0


# Initialising Project 'wailstest'
Project Name      | wailstest
Project Directory | e:\workspace\golang\wailstest\wailstest
Template          | Vanilla + Vite
Template Source   | https://wails.io

Initialised project 'wailstest' in 21.37s.

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony


2、 wails build
DEB | Using go webview2loader
Wails CLI v2.6.0


# Build Options

Platform(s)        | windows/amd64
Compiler           | D:\apps\Go\bin\go.exe
Skip Bindings      | false
Build Mode         | production
Devtools           | false
Frontend Directory | e:\workspace\golang\wailstest\wailstest\frontend
Obfuscated         | false
Skip Frontend      | false
Compress           | false
Package            | true
Clean Bin Dir      | false
LDFlags            |
Tags               | []
Race Detector      | false

# Building target: windows/amd64

  • Generating bindings: Done.
  • Installing frontend dependencies: Done.
  • Compiling frontend: Done.
  • Generating application assets: Done.
  • Compiling application: Done.
 INFO  Wails is now using the new Go WebView2Loader. If you encounter any issues with it, please report them to https://github.com/wailsapp/wails/issues/2004. You could also use the old legacy loader with `-tags native_webview2loader`, but keep in mind this will be deprecated in the near future.
Built 'e:\workspace\golang\wailstest\wailstest\build\bin\wailstest.exe' in 13.651s.

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony

3、Distribution to other users
I use dingtalk to send to others
Antivirus software is triggered at this step
![6261C585-F61E-4e55-B3FB-EC6A44F9D5FD](https://github.com/wailsapp/wails/assets/9362663/90c321ff-24ef-4d23-9fc1-19b655318306)
![983C9382-D325-4e60-8146-D88BC1CA732F](https://github.com/wailsapp/wails/assets/9362663/189e9491-dcd7-4022-ba13-807b59fd6ba0)

**Comment by leaanthony** [comment]
No point leaving this open. If anyone has any ideas what we can do to make Microsoft fix defender then please open another ticket.
